source/elements/oneMKL/source/domains/spblas/operations/spsv.rst

```diff
@@ -0,0 +1,352 @@
+           oneapi::mkl::sparse::matrix_view           A_view,
+           oneapi::mkl::sparse::matrix_handle_t       A_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t x_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t y_handle,
+           oneapi::mkl::sparse::spsv_alg              alg,
+           oneapi::mkl::sparse::spsv_descr_t          spsv_descr,
+           std::size_t                                &temp_buffer_size);
+
+       void spsv_optimize(
+           sycl::queue                                &queue,
+           oneapi::mkl::transpose                     opA,
+           const void*                                alpha,
+           oneapi::mkl::sparse::matrix_view           A_view,
+           oneapi::mkl::sparse::matrix_handle_t       A_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t x_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t y_handle,
+           oneapi::mkl::sparse::spsv_alg              alg,
+           oneapi::mkl::sparse::spsv_descr_t          spsv_descr,
+           sycl::buffer<std::uint8_t, 1>              workspace);
+
+       sycl::event spsv_optimize(
+           sycl::queue                                &queue,
+           oneapi::mkl::transpose                     opA,
+           const void*                                alpha,
+           oneapi::mkl::sparse::matrix_view           A_view,
+           oneapi::mkl::sparse::matrix_handle_t       A_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t x_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t y_handle,
+           oneapi::mkl::sparse::spsv_alg              alg,
+           oneapi::mkl::sparse::spsv_descr_t          spsv_descr,
+           void*                                      workspace,
+           const std::vector<sycl::event>             &dependencies = {});
+
+       sycl::event spsv(
+           sycl::queue                                &queue,
+           oneapi::mkl::transpose                     opA,
+           const void*                                alpha,
+           oneapi::mkl::sparse::matrix_view           A_view,
+           oneapi::mkl::sparse::matrix_handle_t       A_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t x_handle,
+           oneapi::mkl::sparse::dense_vector_handle_t y_handle,
+           oneapi::mkl::sparse::spsv_alg              alg,
+           oneapi::mkl::sparse::spsv_descr_t          spsv_descr,
+           const std::vector<sycl::event>             &dependencies = {});
+
+   }
+
+.. container:: section
+
+   .. rubric:: Notes
+
+   - ``spsv_buffer_size`` and ``spsv_optimize`` must be called at least once
+     before ``spsv`` with the same arguments. ``spsv`` can then be called
+     multiple times. Calling ``spsv_optimize`` on the same descriptor can reset
+     some of the descriptor's data such as the ``workspace``.
+   - The data of the dense handle ``x_handle`` can be reset-ed before each call to ``spsv``.
```

Comment:
The past tense of reset is reset itself. Please use that instead of "reset-ed" here and elsewhere.

Reply:
Thanks, I removed them in https://github.com/uxlfoundation/oneAPI-spec/pull/522/commits/1c236f79e92452c964ab547b847b9d45ad9812eb.